libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -209,13 +210,19 @@ open class SalesforceSDKManager protected constructor(
      */
     private var showDeveloperSupportBroadcastIntentReceiver: BroadcastReceiver? = null
 
-    val webviewLoginActivityClass: Class<out Activity> = loginActivity ?: LoginActivity::class.java
+    val webViewLoginActivityClass: Class<out Activity> = loginActivity ?: LoginActivity::class.java
```

Comment:
I noticed we (inconsistently) use `webview` a lot but the official APIs use `webView` so this seemed like a good time to fix it everywhere.

We also use `frontdoor` instead of `frontDoor` but I'll leave that alone for now 😉.